### PR TITLE
Issue #72:Update README to include Stream methods and other methods

### PR DESCRIPTION
**Pull request by bayern39**
depend on #72

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -149,4 +152,51 @@ aphrodite.addLabelToPatch(patch, "bug");
 // 7.Find patches related with the patch
 List<Patch> patches=aphrodite.findPatchesRelatedTo(patch);
 
+// 8.Retrieve all labels associated with the repository
+Repository repo=new Repository(new URL("https://github.com/bayern39/LearningCoder"));
```
Comment (ryanemerson) [inline README.md]: I would change this repository to "https://github.com/jboss-set/aphrodite_test". You won't want all of the notifications when people try the code for the first time.

File: README.md [hunk 2]
```diff
@@ -149,4 +152,51 @@ aphrodite.addLabelToPatch(patch, "bug");
...
+List<Label> labelss=aphrodite.getLabelsFromPatch(patch);
+
+// 10.delete a label from patch
+aphrodite.removeLabelFromPatch(patch, "label_name");
+
+// 11.Set the labels for the provided patch
+List<Label> labels=new ArrayList<>();
+labels.add(new Label("label_name1"));
+labels.add(new Label("label_name2"));
+labels.add(new Label("label_name3"));
```
Comment (ryanemerson) [inline README.md]: Change these label names to ones that exist at https://github.com/jboss-set/aphrodite_test

File: README.md [hunk 3]
```diff
@@ -149,4 +152,51 @@ aphrodite.addLabelToPatch(patch, "bug");
...
+ CommitStatus status=aphrodite.getCommitStatusFromPatch(patch); 
+ 
+```
+##### stream example
+```java
+// 1.Get all streams from a url
+List<Stream> ss=aphrodite.getAllStreams();
+
+// 2.Get stream by stream's name eg:"wildfly"
+Stream stream=aphrodite.getStream("stream_name");
```
Comment (ryanemerson) [inline README.md]: As we are pulling the jboss-streams file, lets change all of the examples so they will work with this file. So actually put ```getStream("wildfly")```

File: README.md [hunk 4]
```diff
@@ -149,4 +152,51 @@ aphrodite.addLabelToPatch(patch, "bug");
...
+List<Repository> repositories=aphrodite.getDistinctURLRepositoriesFromStreams();
+
+// 4.Retrieve all Repositories associated with a given Stream eg:"wildfly"
+List<Repository> urls=aphrodite.getDistinctURLRepositoriesByStream("stream_name");
+
+// 5.Find all the streams associated to the given repository and codebase
+//repository_url,eg:"https://github.com/aeshell/aesh"
+//code_branch,eg:"master"
+Repository repo=new Repository(new URL("repository_url"));
+Codebase code=new Codebase("code_branch");
```
Comment (ryanemerson) [inline README.md]: Again, lets make these examples that will work. Maybe use wildfly?

File: README.md [hunk 5]
```diff
@@ -60,6 +60,9 @@ RepositoryConfig githubService = new RepositoryConfig("https://github.com/", "yo
 List<RepositoryConfig> repositoryConfigs = new ArrayList<>();
 repositoryConfigs.add(githubService);
 
+StreamConfig streamService=new StreamConfig(new URL("https://raw.githubusercontent.com/jboss-set/jboss-streams/master/streams.json"), StreamType.JSON);
+List<StreamConfig> streamConfigs=new ArrayList<>();
+streamConfigs.add(streamService);
 
 AphroditeConfig config = new AphroditeConfig(issueTrackerConfigs, repositoryConfigs);
```
Comment (ryanemerson) [inline README.md]: This needs to be:

``` 
AphroditeConfig config = new AphroditeConfig(issueTrackerConfigs, repositoryConfigs, streamConfigs);
```

**Comment by ryanemerson** [comment]
@bayern39 You will also need to change ``` issue.setAssignee("ryanemerson");``` to utilise the new User object.

**Comment by bayern39** [comment]
@ryanemerson Fixed!

**Comment by ryanemerson** [comment]
@bayern39 merged, thanks!

**Pull request merged**